src/uct/ib/rc/base/rc_iface.c

```diff
@@ -173,6 +173,35 @@ ucs_status_t uct_rc_iface_query(uct_rc_iface_t *iface,
     return UCS_OK;
 }
 
+ucs_status_t uct_rc_iface_get_address(uct_iface_h tl_iface,
+                                      uct_iface_addr_t *addr)
+{
+    uct_rc_iface_t UCS_V_UNUSED *iface = ucs_derived_of(tl_iface,
```

Comment:
why iface is marked as unused?

Reply:
To avoid compilation warnings, because `UCT_RC_IFACE_TM_ENABLED(iface)` may be defined as `0`, if TM is not configured